demo/build.gradle

```diff
@@ -40,7 +40,6 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.google.android.gms:play-services:4.3.23'
     compile project(':googlemediaframework')
-
+    compile 'com.google.android.gms:play-services:7.5.0'
```

Comment:
Please replace the existing line rather than adding this at the bottom (unless there's some reason why it's now at the bottom)